invenio_access/upgrades/access_2015_05_06_accROLE_accACTION_accARGUMENT_id.py

```diff
@@ -57,6 +59,18 @@ def do_upgrade():
     # table accROLE_accACTION_accARGUMENT
 
     # - drop primary key
+    try:
+        op.execute(
```

Comment:
I have to test it but I am afraid that `op.execute` fails silently, at least with the current one, event if there is no index, the upgrade is marked as done successfully when, in theory, it should fail